### PR TITLE
apply slf4j, logback and lombok

### PR DESCRIPTION
**Pull request by offMomySon**

### CODE REVIEW

File: src/main/java/kr/or/connect/reservation/controller/ReservationController.java [hunk 1]
```diff
@@ -2,30 +2,30 @@
...
+
+@Slf4j
 @Controller
 public class ReservationController {
-	 private static final Logger logger = LoggerFactory.getLogger(ReservationController.class);
 
 	@GetMapping(path = "/htmls/mainpage.html")
 	public String showMainPage(HttpSession session) {
-		logger.info("mainpage.html page called");
+		log.info("mainpage.html page called");
```
Comment (joey-yoonsung) [inline src/main/java/kr/or/connect/reservation/controller/ReservationController.java:16]: API call would be logged as info level by Spring Framework. The debug level would be appropriate on this line.

File: src/main/java/kr/or/connect/reservation/controller/ReservationController.java [hunk 2]
```diff
@@ -2,30 +2,30 @@
...
 		return "mainpage";
 	}
 	
 	@GetMapping(path= "/htmls/detail.html")
 	public String showDetailPage(HttpSession session) {
-		logger.info("detail.html page called");
+		log.info("detail.html page called");
```
Comment (joey-yoonsung) [inline src/main/java/kr/or/connect/reservation/controller/ReservationController.java:22]: ditto

File: src/main/java/kr/or/connect/reservation/controller/ReservationController.java [hunk 3]
```diff
@@ -2,30 +2,30 @@
...
 		return "detail";
 	}
 	
 	@GetMapping(path= "/htmls/reserve.html")
 	public String showReservePage(HttpSession session) {
-		logger.info("reserve.html page called");
+		log.info("reserve.html page called");
```
Comment (joey-yoonsung) [inline src/main/java/kr/or/connect/reservation/controller/ReservationController.java:28]: ditto

File: src/main/java/kr/or/connect/reservation/controller/ReservationController.java (hunk 1 above)
Comment (offMomySon) [reply src/main/java/kr/or/connect/reservation/controller/ReservationController.java]: i fix that by force push,
is there any way fixing in this page?

File: src/main/java/kr/or/connect/reservation/controller/ReservationController.java (hunk 2 above)
Comment (offMomySon) [reply src/main/java/kr/or/connect/reservation/controller/ReservationController.java]: ditto

File: src/main/java/kr/or/connect/reservation/controller/ReservationController.java (hunk 3 above)
Comment (offMomySon) [reply src/main/java/kr/or/connect/reservation/controller/ReservationController.java]: ditto

**Pull request merged**